install/linux/usr/share/odemis/sim/meteor-sim.odm.yaml

```diff
@@ -0,0 +1,203 @@
+               "Linked Stage", "EBeam",
+    ],
+}
+
+"SEM": {
+    class: simsem.SimSEM,
+    role: null,
+    init: {
+           image: "simsem-fake-output.h5", # any large 16 bit image is fine
+    },
+    children: {scanner: "EBeam",
+               detector0: "SE Detector",
+               focus: "EBeam Focus",
+    }
+}
+
+"EBeam": {
+    # Internal child of SimSEM, so no class
+    role: e-beam,
+    init: {},
+    affects: ["BSED Phenom", "Camera"], # affects the CCD in case of cathodoluminescence
+}
+
+"EBeam Focus": {
+    role: ebeam-focus,
+    init: {},
+    affects: ["EBeam"]
+}
+
+"SE Detector": {
+    # Internal child of SimSEM, so no class
+    role: se-detector,
+    init: {},
+}
+
+# Normally provided by the SEM
+"Stage": {
+    class: tmcm.TMCLController,
+    role: stage-bare,
+    init: {
+        port: "/dev/fake6",
+        address: null,
+        axes: ["x", "y", "z", "rx", "rz"],
+        ustepsize: [1.e-7, 1.e-7, 1.e-7, 1.2e-5, 1.2e-5], # unit/µstep
+        rng: [[-0.1, 0.1], [-0.05, 0.05], [0, 0.1], [-2, 2], [0, 6.28]],
+        unit: ["m", "m", "m", "rad", "rad"],
+        refproc: "Standard",
+    },
+    metadata: {
+        FAV_POS_DEACTIVE: { 'rx': 0, 'rz': 1.9076449, 'x': -0.01529, 'y': 0.0506, 'z': 0.01975 },
+        SEM_IMAGING_RANGE: {"x": [-0.0049, 0.0048], "y": [0.0004, 0.0044], "z": [0.0055, 0.0075]},
+        FM_IMAGING_RANGE: {"x": [0.044, 0.054], "y": [-0.0044, 0.0045], "z": [0.0046, 0.0087]},
+        SAMPLE_CENTERS: {"GRID 1": {'x': -0.003, 'y':  0.0025, 'z': 0.0066}, "GRID 2": {'x': 0.003, 'y': 0.0023, 'z': 0.0066}},
+        # Note that these values change every session 
+        POS_COR: { "x": 0.02447, "y": -0.000017 },
+        # Active tilting & rotation angles positions when switching between SEM & FM
+        # Note: these values are calibrated at installation time 
+        FAV_FM_POS_ACTIVE: {"rx": 0.122173048  , "rz":  5.06145483},
+        FAV_SEM_POS_ACTIVE: {"rx": 0.34906585, "rz": 1.91986218}    # Note that milling angle (rx) can be changed per session 
+    },
+}
+
+"Linked Stage": {
+    class: actuator.LinkedAxesActuator,
+    role: stage,
+    dependencies: {
+        "stage": "Stage",
+    },
+    affects: ["Camera", "EBeam"],
+    metadata: {
+        # Matrix to convert from X/Y to X/Y/Z (on the underlying stage)
+        # xd = POS_COR[0] + CALIB[0,0] * x + CALIB[0,1] * y
+        # yd = POS_COR[1] + CALIB[1,0] * x + CALIB[1,1] * y
+        # zd = POS_COR[2] + CALIB[2,0] * x + CALIB[2,1] * y
+        CALIB: [[1, 0],  [0, 0.6941955197522921], [0, -0.7197864824764667]],
+        POS_COR: [0, -0.0007363771884364807, 0.004945901888925916],
+        # Allowed range when in FM imaging mode. The X/Y ranges are also used
+        # as default area for the overview acquisition. The values must be in the
+        # stage coordinates, so converted from the SEM stage (in Y).
+        POS_ACTIVE_RANGE: {"x":  [50.92e-3, 53.45e-3], "y":  [-4.3e-3, -1.82e-3]},
```

Comment:
ACTIVE_RANGE is used by the GUI to limit the area where the user is allowed to go when dragging/moving the image. It has to be on the component with role=`stage`. 
The FM_IMAGING_RANGE is on the stage-bare. I think you could also use ACTIVE_RANGE there, as it would make sense to have both simultaneously. I'm also fine with leaving FM_IMAGING_RANGE as it's clearer what it is about.